src/FrameBuffer.cpp

```diff
@@ -677,7 +677,8 @@ void FrameBufferList::setBufferChanged(f32 _maxY)
 void FrameBufferList::clearBuffersChanged()
 {
 	gDP.colorImage.changed = FALSE;
-	FrameBuffer * pBuffer = frameBufferList().findBuffer(*REG.VI_ORIGIN & 0xffffff);
+	//FrameBuffer * pBuffer = frameBufferList().findBuffer(*REG.VI_ORIGIN & 0xffffff);
```

Comment:
The original Gliden64 behavior should be supported.  Use the #ifdef NATIVE pre processor flag to signify native or original mode.